### PR TITLE
Fix sanity pylint

### PR DESCRIPTION
**Pull request by alagoutte**

### CODE REVIEW

File: plugins/module_utils/arubaoss.py [hunk 1]
```diff
@@ -300,6 +306,8 @@ def run_commands(self, uri, payload={}, method="POST",
         '''
         reboot = None
         response = None
+        if payload is None:
+            payload = []
```
Comment (tchiapuziowong) [inline plugins/module_utils/arubaoss.py:310]: `payload` should be of type dict not list correct?

File: plugins/module_utils/arubaoss.py [hunk 2]
```diff
@@ -113,13 +113,16 @@ def __init__(self, module):
...
+        self._url = "{1}://{2}:{3}/rest/{4}".format(proto, host, port, api)
 
-    def _send(self, url, method='POST', body={}):
+    def _send(self, url, method='POST', body=None):
         '''Sends command to device '''
 
         headers = {'Content-Type': 'application/json'}
 
+        if body is None:
+            body = []
```
Comment (tchiapuziowong) [inline plugins/module_utils/arubaoss.py:124]: `body` should be of type dict not list correct?

File: plugins/module_utils/arubaoss.py [hunk 3]
```diff
@@ -248,13 +251,16 @@ def __init__(self, module):
...
```
Comment (tchiapuziowong) [inline plugins/module_utils/arubaoss.py:262]: `body` should be of type dict not list correct?

**Review by tchiapuziowong — COMMENTED** [review]
Please see comments - can you validate that the modules function as expected with the changes provided in this PR?

**Comment by alagoutte** [comment]
Oups typo (wrong copy/paste) -> fixed

**Comment by tchiapuziowong** [comment]
Thank you for the contribution! At this time I have other obligations but I will review at my earliest convenience!

**Comment by alagoutte** [comment]
> Thank you for the contribution! At this time I have other obligations but I will review at my earliest convenience!

Sure ! no problem !

**Comment by alagoutte** [comment]
Any news ?

**Comment by alagoutte** [comment]
bump ? @tchiapuziowong  @Keeper-of-the-Keys @karthikeyan-dhandapani

**Comment by Keeper-of-the-Keys** [comment]
@alagoutte I'm not able to do a review for you, I don't work for HP/Aruba, I also no longer work at a company that has Aruba equipment so can't check the changes in a real world environment.

But if it helps the changes do look OK to me...

**Comment by alagoutte** [comment]
> @alagoutte I'm not able to do a review for you, I don't work for HP/Aruba, I also no longer work at a company that has Aruba equipment so can't check the changes in a real world environment.
> 
> But if it helps the changes do look OK to me...

Oups sorry wrong ping mention !

**Review by tchiapuziowong — APPROVED** [review]

**Comment by tchiapuziowong** [comment]
I'm working on publishing some fixes that development has been working on and I'll include this in the next release - goal is by end of week next week.

**Comment by alagoutte** [comment]
> I'm working on publishing some fixes that development has been working on and I'll include this in the next release - goal is by end of week next week.

Perfect,

a new series of patch
* https://github.com/aruba/aos-switch-ansible-collection/pull/59
* https://github.com/aruba/aos-switch-ansible-collection/pull/60
* https://github.com/aruba/aos-switch-ansible-collection/pull/61
* https://github.com/aruba/aos-switch-ansible-collection/pull/62
* https://github.com/aruba/aos-switch-ansible-collection/pull/63

**Comment by SalutAToi** [comment]
Hey @alagoutte ! All the fixes for `sanity: fix pylint ansible-format-automatic-specification` have wrong indexes (starting at 1 instead of 0), which breaks it
https://github.com/aruba/aos-switch-ansible-collection/commit/45f1379aa66b635d08f2e0fb28e95b365f913a73

**Comment by alagoutte** [comment]
> Hey @alagoutte ! All the fixes for `sanity: fix pylint ansible-format-automatic-specification` have wrong indexes (starting at 1 instead of 0), which breaks it [45f1379](https://github.com/aruba/aos-switch-ansible-collection/commit/45f1379aa66b635d08f2e0fb28e95b365f913a73)

Oups, i will push a fix...

**Pull request merged**